lib/components/ContextPanel/ContextPanel.module.scss

```diff
@@ -19,8 +19,8 @@ $button-width: 30*$grid-size;
         width: 100%;
     }
 
-    background-color: var(--color-bg-container-panel);
-    box-shadow: var(--shadow-container-panel);
+    background-color: var(--color-content-background-primary);
+    box-shadow: 0 25.6px 57.6px 0 var(--color-dark-transparent-tenth);
```

Comment:
lol why these numbers? Can't we standardize on a set of box shadow variables?

Reply:
this where the values that were there before in `var(--shadow-container-panel)` I just moved them here to standarize var uses to just colors and not the whole box-shadow.

In theory there is a set of standard box-shadows (depth variables), I can take a look at using those here.